ASUWebPlatforms/webspark-ci
# WS2-1847 : Check for/add missing horizontal cards

### Description

We have some horizontal cards, such as the horizontal story card which aren’t represented in WS2. For this ticket:

take stock of what’s missing

implement the missing items - we can discuss post-scrum the best approach for implementing the components. 

To be added as individual cards like the Card Image and Content component (instead of arrangements which force columns that don’t make sense for  horizontal cards). 

Solution :
Creating new functionality to allow horizontal display for some Card arrangement options


**QA Steps:**

1. Add a new “Card Arrangement” block

2. Ensure the presence of the recently added feature "Display orientation."

. If you choose “Add Card Group Ranking” or “Add Image based cards” then “Display orientation”  must be hidden. That option won’t be displayed for those two options.

.  If you choose another Card type and  then “Display orientation = Horizontal”  the  “Columns to Display” select have only “Two Columns” option.


### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-1847)

### Checklist


### Verified in browsers 


### Screenshots



Note: Sections that are not applicable for this issue can be removed.
